integreat_cms/api/urls.py

```diff
@@ -123,6 +128,28 @@
 urlpatterns = [
     path("api/regions/", include(region_api_urlpatterns)),
     path("wp-json/extensions/v3/sites/", include(region_api_urlpatterns)),
+    path(
+        "api/social/",
+        include(
+            [
+                path(
+                    "<slug:region_slug>/<slug:language_slug>/<path:path>/",
+                    page_socialmedia_headers,
+                    name="page_socialmedia_headers",
+                ),
+                path(
+                    "<slug:region_slug>/<slug:language_slug>/",
+                    region_socialmedia_headers,
+                    name="region_socialmedia_headers",
+                ),
```

Comment:
I think we should store the web app's reserved routes in our constants anyway, e.g. to be able to prevent regions to be created with the slug "landing" (also see #1792), so maybe this can be resolved together? If it's too complicated, we could also merge this PR fist and open another issue for improving this in the future.